curriculum/updated-curriculum/html/basic-html-07.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h2>CatPhotoApp</h2>
+    <main>
+    <!-- TODO: Add link to cat photos -->
+    <p>Click here to view more cat photos.</p>
+	</main>
+  </body>
+</html>
+
+<!-- HTML elements are often nested within other HTML elements. On this page, a comment and a p element are nested within the main element. When you nest an element inside another element, the nested element becomes a child of the parent. To make HTML easier to read, you should indent child elements inside their parent element. Indent the comment and p element on this page so it is easer to see that they are children of the main element. (Note: Indentation, spacing and new lines do not impact how a browser renders a page but can make it easier to read HTML.) -->
```

Comment:
"HTML elements are often nested within other HTML elements." I recommend avoiding passive voice like this, as it sounds boring. Also, this test text is way longer than it needs to be. 

Even though I recommend removing the <main> element, here's how I would reword this test:

<!-- When you put one HTML element inside of another HTML element ("nesting"), indent it so it's easier to read. Note that adding spaces in front of your HTML just changes how code looks - not how it runs. Add spaces in front of the <p> element so it's clear that it's nested inside the <main> element. -->